### DISCUSSION TITLE
default notification daemon/center

### DISCUSSION THREAD

**Discussion by denisok** [openSUSE/openSUSEway#81] [Ideas]
Hi here!

currently openSUSEway uses `mako` as a notification daemon, it gets installed with pattern-sway.

There is a contribution to Factory and openSUSEWay by @FilippoBonazziSUSE:
- https://github.com/openSUSE/openSUSEway/pull/76
- https://build.opensuse.org/package/show/openSUSE:Factory/SwayNotificationCenter

And it looks good and has nifty features.

What do you think about replacing mako with swaync for openSUSEway?

If that would be an option, it would be interesting to find out how to replace mako with it, as it probably already installed on many systems.

**Comment by FilippoBonazziSUSE** [comment]
Good point about the replacing.  We could probably add swaync and its configuration, while leaving the mako configuration in on the side for one release, notify users (e.g. in the Release Notes) and remove the mako configuration and spec file dependency in a future release.

**Comment by FilippoBonazziSUSE** [comment]
Any other suggestions on how to go about this?

So that we can close #62.

**Comment by denisok** [comment]
during the testing I found that we could just exec swaync in our conf. I will submit PR shortly.

**Reply by FilippoBonazziSUSE** [reply]
swaync comes with a provided systemd user service. I would rather start it with that. the service is disabled by default, we could enable it when installing openSUSEway

**Reply by FilippoBonazziSUSE** [reply]
What we really should do is remove the openSUSEway dependency on `mako` - it sometimes gets started before `swaync` and breaks it. I'm not sure how to disable it since it is started over DBUS, and I can't remove it since it will pull all of openSUSEway and sway...

**Reply by denisok** [reply]
right, both have D-Bus services. I haven't found so far how D-Bus prioritizes which one should be picked up.

**Reply by FilippoBonazziSUSE** [reply]
Also I found that `mako` is a strict dependency of the `patterns-sway` package, together with a few others. I was thinking we could remove that dependency, and possibly add `SwayNotificationCenter` as a Recommend rather than a strict dep... opinions?

**Reply by denisok** [reply]
yes - please do

**Comment by FilippoBonazziSUSE** [comment]
I will close this for now since I think we're quite happy with SwayNotificationCenter
